### DISCUSSION TITLE
Use SwiftSoup for parse

### DISCUSSION THREAD

**Discussion by keetsta** [SubvertDev/ForPDA#22] [General]
I beg you

**Comment by SubvertDev** [comment]
I used it before when I parsed 4pda pages as html. But now we have another type of connection, which is not html anymore, but nested arrays of data (**not** JSON). If you have any ideas to handle it in a more robust way, feel free to open PR!

I'll convert this to a discussion. You can add any suggestions there or send a message in telegram channel [here](t.me/forpda_ios_chat)!
